portfolio/src/main/webapp/misc.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Socials</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="topnav">
+      <a class="active" href="index.html">Aaron Lopes</a>
+      <a href="projects.html">projects</a>
+      <a href="socials.html">socials</a>
+      <a href="misc.html">misc</a>
+    </div>
+  </body>
+</html>
```

Comment:
You'll want to keep the last new-line at the end, which is conventional. For this file and the others. Sorry for the confusion!

Reply:
No worries, forgot about the convention on my end.